packages/flutter_tools/test/general.shard/build_system/build_system_test.dart

```diff
@@ -319,8 +319,7 @@ void main() {
       fs.file('a.txt').writeAsStringSync('a');
       called += 1;
```

Comment:
```suggestion
      called++;
```

Reply:
See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-over